src/sentry/models/activity.py

```diff
@@ -13,6 +16,58 @@
 from sentry.tasks import activity
 from sentry.types.activity import CHOICES, ActivityType
 
+if TYPE_CHECKING:
+    from sentry.models import Group, User
```

Comment:
Why does this have to be in the `if TYPE_CHECKING`?

Reply:
```
INTERNALERROR> ImportError: cannot import name 'Group'
```
Yup, still needed 😐